weblogic-azure-aks/src/main/arm/scripts/inline-scripts/validateParameters.sh

```diff
@@ -490,23 +551,23 @@ sslConfigurationAccessOption=$9
 appGatewayCertificateOption=${10}
 enableAppGWIngress=${11}
 checkDNSZone=${12}
-checkACR=${13}
 
 ocrLoginServer="container-registry.oracle.com"
+ocrGaImagePath="middleware/weblogic"
+ocrCpuImagePath="middleware/weblogic_cpu"
 sslCertificateKeyVaultOption="keyVaultStoredConfig"
 userManagedIdentityType="Microsoft.ManagedIdentity/userAssignedIdentities"
+gitUrl4CpuImages="https://raw.githubusercontent.com/oracle/weblogic-azure/main/weblogic-azure-aks/src/main/resources/weblogic_cpu_images.json"
```

Comment:
This appears to be a DRY violation. I see this same name=value in `common.sh`.  Can we source it from there?

Reply:
Yes, Ed, I just found we can combine context of files in Deployment Script by `loadTextContent` of each file:

```
scriptContent: format('{0}\r\n\r\n{1}', loadTextContent('../../../arm/scripts/common.sh'), loadTextContent('../../../arm/scripts/inline-scripts/validateParameters.sh'))
```

see https://github.com/oracle/weblogic-azure/pull/104/commits/65a1d4b70cf9c09cf92b31d0d6d9f98164120009